uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/request-ctrl_test.js

```diff
@@ -0,0 +1,158 @@
+/* eslint-env amd, jasmine */
```

Comment:
We should not create a test file for `request-ctrl.js`as it is a parent class and never used directly. All the tests for this class should go to `leave-request-ctrl_test`.